controllers/storagecluster/rbd_mirror_debug.go

```diff
@@ -0,0 +1,129 @@
+// This is a temporary measure to enable verbose logging for RDR
+// This should be removed once RDR goes to GA.
+package storagecluster
+
+import (
+	"context"
+	"os"
+
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+)
+
+func (r *StorageClusterReconciler) ensureRbdMirrorDebug(sc *ocsv1.StorageCluster) error {
+	if sc.Spec.Mirroring.Enabled {
```

Comment:
If the user disables rbd mirroring, should we revert the settings that had been enabled? Or are we assuming these settings will remain enabled for the lifetime of the cluster until we GA? Or they would just need to use the toolbox to disable them manually.

Reply:
These settings should be mostly harmless to leave on but it would be nice to disable them automatically -- it's just a bunch of `ceph config rm` commands after all.

Reply:
Sounds good, I will create another job that will be run when mirroring is disabled. It will remove the settings we applied with the creation job.